### DISCUSSION TITLE
When I close the quick-fix window, focus moves to nvim-tree even if another buffer is open

### DISCUSSION THREAD

**Discussion by johnmathews** [nvim-tree/nvim-tree.lua#1927] [Q&A]
I would like to prevent nvim-tree being focussed when I close the quick-fix window. Is this possible? 

It seems like focus always moves the left most buffer, which is kind of annoying as I usually want to continue working in the buffer I was in when I opened the quick-fix window.

**Comment by alex-courtis** [comment]
A quick test with a [minimal configuration](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=bug&template=bug_report.yml) resulted in a file buffer window being focused on `:cclose` or `:q`, regardless of whether nvim-tree was focused before.

Do you have an other plugins/automation that might be affecting this behaviour?

**Reply by johnmathews** [reply]
Thanks for testing!

I tested it without "Asheq/close-buffers.vim" and "marklcrns/vim-smartq" and the behavior is the same. I had thought that it was because (neo)vim will focus the left most buffer by default, but if your test shows otherwise then it looks like that is also false.

In your test, was the layout like below? or was perhaps the qf window on the right?

```text
-----------------------------------
|            |                    |
|            |                    |
|  nvim-tree |     buffer A       |
|            |                    |
|            |                    |
|            |                    |
|            |                    |
-----------------------------------
|                                 |
|          quick-fix              |
|                                 |
-----------------------------------
```

**Reply by alex-courtis** [reply]
I'm afraid that's just vim behaviour, nothing to do with nvim-tree. The buffer above will be focused on :cclose.

See `belowright` vs `botright`:
[![asciicast](https://asciinema.org/a/553358.svg)](https://asciinema.org/a/553358)
